specification/e-referrals-service-api.yaml

```diff
@@ -496,7 +496,44 @@ paths:
           $ref: 'components/schemas/responses/Forbidden.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'
+  /R4/v1/PractitionerRole:
```

Comment:
It's not ideal that there are different names for the same (?) thing depending on where you look :
"role", "practitioner role", "business function"

But I guess this is a historical accident and not something we can change.